researchapp/fixtures.yml

```diff
@@ -30,14 +30,14 @@ class: !!python/name:researchapp.models.providers.OAuthClient
 args:
   client_id: research-app
   client_secret: demo-secret-s4s
-  scope: launch/patient patient/*.read offline_access
+  scope: patient/*.read offline_access
```

Comment:
fhirclient adds that automatically. https://github.com/smart-on-fhir/client-py/blob/43e9f12b2507c11e2afbe55fa2f6c4bcac7c68cb/fhirclient/client.py#L91